.semaphore/semaphore.yml

```diff
@@ -0,0 +1,87 @@
+# Use the latest stable version of Semaphore 2.0 YML syntax:
+  version: v1.0
+
+  name: Simple.org CI
+
+  # An agent defines the environment in which your code runs.
+  # It is a combination of one of available machine types and operating
+  # system images.
+  # See https://docs.semaphoreci.com/article/20-machine-types
+  # and https://docs.semaphoreci.com/article/32-ubuntu-1804-image
+  agent:
+    machine:
+      type: e1-standard-2
+      os_image: ubuntu1804
+
+  # Blocks are the heart of a pipeline and are executed sequentially.
+  # Each block has a task that defines one or more jobs. Jobs define the
+  # commands to execute.
+  # See https://docs.semaphoreci.com/article/62-concepts
+  blocks:
+    - name: Setup
+      task:
+        jobs:
+          - name: yarn
+            - nvm use 8.11.3
+            - sudo touch /etc/apt/sources.list.d/google-chrome-source.list
+            - install-package --update-new google-chrome-stable-
+            - yarn install
+          - name: bundle
+            commands:
+            # See https://docs.semaphoreci.com/article/54-toolbox-reference#checkout
+            - checkout
+            # Restore dependencies from cache.
+            # Read about caching: https://docs.semaphoreci.com/article/149-caching
+            - cache restore
+            # Set Ruby version:
+            - sem-version ruby 2.7.1
+            - bundle install --deployment -j 4 --path vendor/bundle
+            - cache store
+            - nvm use 8.11.3
+            # - sudo touch /etc/apt/sources.list.d/google-chrome-source.list
+            # - install-package --update-new google-chrome-stable-
+            - yarn install
+
+    - name: Unit tests
+      task:
+        # This block runs two jobs in parallel and they both share common
+        # setup steps. We can group them in a prologue.
+        # See https://docs.semaphoreci.com/article/50-pipeline-yaml#prologue
+        prologue:
+          commands:
+            - checkout
+            - cache restore
+            - sem-service start postgres
+            - sem-version ruby 2.7.1
+            - bundle install --deployment --path vendor/bundle
+            - bundle exec rake db:setup
+            - bundle exec rake db:test:prepare
+
+        jobs:
+        - name: RSpec - model tests
+          commands:
+            - bundle exec rspec spec/models
+
+        - name: RSpec - controller tests
+          commands:
+            - bundle exec rspec spec/controllers
```

Comment:
I think we are missing a bunch of our specs with this approach, like api / service / queries.